### PR TITLE
Add descriptions to scheduled tasks

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

A while ago I opened an issue on `sidekiq-cron` repo to request a description field. Apparently it's already there even though I couldn't find it mentioned in the README

## Related Tickets & Documents

https://github.com/ondrejbartas/sidekiq-cron/issues/294

## QA Instructions, Screenshots, Recordings

1. Add `tech_admin` to your roles, eg. `user.add_role(:tech_admin)`
2. Go to http://localhost:3000/sidekiq/cron and select any of the tasks, eg. http://localhost:3000/sidekiq/cron/award_beloved_comment_badges
3. See the description in the page:

Screenshot_2021-01-20  DEVELOPMENT  Sidekiq

## Added tests?

- [ ] Yes
- [x] No, and this is why: only a documentation update
- [ ] I need help with writing tests

### CODE REVIEW

File: config/schedule.yml [hunk 1]
```diff
@@ -1,113 +1,139 @@
...
 record_daily_usage:
+  description: "Records daily usage stats (runs daily at 11:00 UTC)"
   cron: "0 11 * * *" # daily at 11:00 UTC
   class: "Metrics::RecordDailyUsageWorker"
 record_daily_notifications:
+  description: "Records daily notifications stats (runs daily at 11:00 UTC)"
   cron: "0 11 * * *" # daily at 11:00 UTC
   class: "Metrics::RecordDailyNotificationsWorker"
 record_data_counts:
+  description: "Records the size of the most import database tables (runs hourly on the 10th minute after the hour)"
```
Comment (Ridhwana) [inline config/schedule.yml:18]: @rhymes this description doesn't make sense, is it `important database`?

File: config/schedule.yml [hunk 2]
```diff
@@ -1,113 +1,139 @@
...
   cron: "10 * * * *" # every hour, 10 min after the hour
   class: "Metrics::RecordDataCountsWorker"
 check_data_update_script_statuses:
+  description: "Records the statuses of data update scripts (runs hourly on the 30th minute after the hour)"
   cron: "30 * * * *" # 30 min after every hour
```
Comment (Ridhwana) [inline config/schedule.yml:23]: Do we need these comments now that we have a description?

File: config/schedule.yml [hunk 3]
```diff
@@ -1,113 +1,139 @@
...
   args:
     - "/feed.xml"
 hourly_badge_cache_bust:
+  description: "Busts the edge cache of badges (runs hourly on the hour)"
   cron: "0 * * * *" # hourly on the hour
   class: "BustCachePathWorker"
   args:
     - "/badge"
 daily_home_cache_bust:
+  description: "Busts the edge cache of the homepage (runs hourly on the hour)"
```
Comment (Ridhwana) [inline config/schedule.yml:114]: Base on the comment below, should be `runs hourly on the hour`  or `runs daily at 12am UTC`?

**Review by Ridhwana — COMMENTED** [review]
@rhymes just left some small comments.

File: config/schedule.yml (hunk 1 above)
Comment (rhymes) [reply config/schedule.yml:18]: "record the size of the most important tables in the database" - this what I meant :D

File: config/schedule.yml (hunk 2 above)
Comment (rhymes) [reply config/schedule.yml:23]: Good call, I can remove those :)

File: config/schedule.yml (hunk 3 above)
Comment (rhymes) [reply config/schedule.yml:114]: You're right, it runs only at midnight UTC. Let me update that, too much copying and pasting :D

**Review by Ridhwana — APPROVED** [review]

**Review by benhalpern — APPROVED** [review]

**Pull request merged**